kodi-stable/snapcraft.yaml

```diff
@@ -0,0 +1,208 @@
+name: kodi  # the name of the snap
+version: "16.1-Jarvis"  # the version of the snap, which can be any string
+summary: Kodi Media Center  # 79 char long summary
+description: |
+  Kodi, formerly known as XBMC Media Center, is an award winning free and
+  open source software media-player and entertainment hub for all your digital
+  media. Kodi is available for Linux, Mac OS X (Leopard, Tiger and Apple TV)
+  and Microsoft Windows, as well as the original Xbox game console. Created in
+  2003 by a group of like minded programmers, Kodi is a non-profit project run
+  and developed by volunteers located around the world. More than 50 software
+  developers have contributed to Kodi, and 100-plus translators have worked to
+  expand its reach, making it available in more than 30 languages.
+confinement: devmode  # start with devmode, then use "strict" to enforce system
+                      # access only via declared interfaces
+
+apps:
+  kodi:
+    command: wrapper_kodi.sh
+    plugs:
+      - unity7
+      - network
+      - network-bind
+      - home
+      - opengl
+      - pulseaudio
+      - mount-observe
+      - optical-drive
+      - camera
+  kodi-standalone:
+    command: wrapper_kodi-standalone.sh
+    plugs:
+      - unity7
+      - network
+      - network-bind
+      - home
+      - opengl
+      - pulseaudio
+      - mount-observe
+      - optical-drive
+      - camera
+
+parts:
+  kodi:
+    source: https://github.com/xbmc/xbmc.git
+    source-tag: 16.1-Jarvis
+    plugin: x_kodiautotools
+    configflags:
+      - --prefix=/
+    build-packages:
```

Comment:
Can you apply this? http://paste.ubuntu.com/21252024/

It add's a few `build-packages`.